### PR TITLE
Create login, logout and register

### PR DESCRIPTION
**Pull request by ghost**
## Description
Create login, logout and register
## Reference Requirements
- Task ID: TI-31

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-config/auth
-views/auth
-controller/auth

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-nale-dn — COMMENTED** [review]
LGTM

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@extends('layouts.app')
```
Comment (at-huydq) [inline resources/views/home.blade.php]: ko cần page này, dùng admin.index dc rồi

File: routes/web.php [hunk 2]
```diff
@@ -14,10 +14,19 @@
...
     Route::get('/', function () {
         return view('admin.index');
     });
     Route::resource('users', 'UserController');
     Route::resource('books', 'BookController');
 });
+
+Auth::routes();
+
+Route::get('/home', 'HomeController@index')->name('home');
```
Comment (at-huydq) [inline routes/web.php]: ko cần route này nha

File: routes/web.php [hunk 3]
```diff
@@ -14,10 +14,19 @@
     return view('welcome');
 });
 
-Route::group(['namespace' => 'Admin', 'prefix' => 'admin', "as" => "admin." ], function(){
+$admin_config = [
+    'namespace' => 'Admin',
+    'prefix' => 'admin',
+    'as'=> 'admin.',
+];
+Route::group($admin_config, function(){
```
Comment (at-huydq) [inline routes/web.php]: thêm middleware auth vào đây nha

File: app/Http/Controllers/HomeController.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
```
Comment (at-huydq) [inline app/Http/Controllers/HomeController.php]: ko cần controller này.

**Review by ghost — COMMENTED** [review]
pls check my cmt

File: routes/web.php (hunk 3 above)
Comment (at-haytran) [reply routes/web.php]: @at-huydq  cái đó em để pull request sau đó anh

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**